.github/workflows/tests.yml

```diff
@@ -1,27 +1,106 @@
 ---
-name: Rust
+name: Tests
 on:
   push:
     branches: [main]
   pull_request:
     branches: [main]
 
-env:
-  CARGO_TERM_COLOR: always
-  ACTIONS_LINTS_TOOLCHAIN: 1.50.0
-
 jobs:
-  linting:
-    name: "Lints, pinned toolchain"
+  tests:
+    # Distro configuration matrix
+    #
+    # Each build is run in a Docker container specific to the distro.
+    # When adding a new distro, handle the dependency installation in
+    # `ci/gh-install.sh`. The matrix configuration options are:
+    #
+    # name: A friendly name to use for the job.
+    #
+    # image: The Docker image to use.
+    #
+    # pre-checkout-setup: Commands to run before the git repo checkout.
+    #   If git is not in the Docker image, it must be installed here.
+    #   Otherwise, the checkout action uses the GitHub REST API, which
+    #   doesn't result in an actual git repo. A real git repo is
+    #   required to checkout the submodules.
+    #
+    # extra-packages: Packages to install in addition to those in
+    #   `ci/gh-install.sh`. This can be used to support features from
+    #   additional `configure` options.
+    #
+    # configure-options: Options to pass to `configure`.
+    strategy:
+      matrix:
+        include:
+          - name: Debian Buster (10) with sign-ed25519
```

Comment:
Maybe use Debian bullseye (11), which is currently in freeze and will hopefully be released on 2021-07-31, for one of the Debian builds?